scripts/build_mirror_config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Generate the sources.list.<arch> in the config path
+
+DEFAULT_MIRROR_URLS=http://debian-archive.trafficmanager.net/debian/,http://cdn1.packages.trafficmanager.net/snapshot/debian/latest/
+DEFAULT_MIRROR_SECURITY_URLS=http://debian-archive.trafficmanager.net/debian-security/,http://cdn1.packages.trafficmanager.net/snapshot/debian-security/latest/
```

Comment:
Need fix for repr build?
Now REPR_MIRROR_URL_PATTERN='http:\/\/packages.trafficmanager.net\/debian'

Reply:
Yes, we need to support repr build, fixed, thanks.